.gitconfig

```diff
@@ -12,5 +12,4 @@
 	allowed = scripts/deploy-backend.sh:.+:DJANGO_SECRET_KEY=..python -c .from secrets import token_urlsafe. print.token_urlsafe..*
 	allowed = .git/config:.*
 	allowed = .gitconfig:.*
-	allowed = .*DJANGO_SECRET_KEY=.*tdp-dev-insecure
-    allowed = ./tdrs-frontend/node_modules/.*
+	allowed = .*DJANGO_SECRET_KEY=.*
```

Comment:
@ADPennington Do you remember why this was in here?

Reply:
i think this was something @n0remac changed to help resolve [this issue](https://app.circleci.com/pipelines/github/ADPennington/TANF-app/85/workflows/ba629731-f090-4979-ba57-5da157d803b7) while testing the prod logingov env vars. i think it was because my master branch was outdated. but @n0remac please confirm.

Reply:
I don't think this was me.

Reply:
@n0remac please see here: https://app.circleci.com/pipelines/github/ADPennington/TANF-app/86/workflows/1906a77b-393e-47ac-b4da-c534f2cc8ff6